### PR TITLE
Update help text to avoid confusion of where files can be added from

### PR DESCRIPTION
**Pull request by diasdavid**
License: MIT
Signed-off-by: David Dias daviddias.p@gmail.com

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 46b34e2836434c14b13db6617ad8dd847932e53d
  * Invalid signoff. Commit message must end with  
    **License: MIT  
Signed-off-by: .* <.*>**

[Guidelines](https://github.com/ipfs/community/blob/master/contribution-guidelines.md#commit-messages) and [a script](https://github.com/ipfs/community/blob/master/dev/tools/hooks/setup_commit_msg_hook.sh) are available to help. Your feedback on GitCop is welcome on [this issue](https://github.com/ipfs/infrastructure/issues/23).

---
This message was auto-generated by https://gitcop.com

File: core/commands/files/files.go [hunk 1]
```diff
@@ -124,7 +124,7 @@ var FilesCpCmd = &cmds.Command{
 		Tagline: "copy files into mfs",
 	},
 	Arguments: []cmds.Argument{
-		cmds.StringArg("source", true, false, "source object to copy"),
+		cmds.StringArg("source", true, false, "source object to copy (e.g. '/ipfs/Qm...'"),
```
Comment (whyrusleeping) [inline core/commands/files/files.go]: it can be either `/ipfs/QmAbc..` or an absolute path within mfs itself.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 46b34e2836434c14b13db6617ad8dd847932e53d
  * Invalid signoff. Commit message must end with  
    **License: MIT  
Signed-off-by: .* <.*>**

 * Commit: a2ade40c9d21d001b795820793df176abbed5b81
  * Invalid signoff. Commit message must end with  
    **License: MIT  
Signed-off-by: .* <.*>**

[Guidelines](https://github.com/ipfs/community/blob/master/contribution-guidelines.md#commit-messages) and [a script](https://github.com/ipfs/community/blob/master/dev/tools/hooks/setup_commit_msg_hook.sh) are available to help. Your feedback on GitCop is welcome on [this issue](https://github.com/ipfs/infrastructure/issues/23).

---
This message was auto-generated by https://gitcop.com

**Comment by jbenet** [comment]
@diasdavid run https://github.com/ipfs/community/blob/master/dev/tools/hooks/setup_commit_msg_hook.sh

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 46b34e2836434c14b13db6617ad8dd847932e53d
  * Invalid signoff. Commit message must end with  
    **License: MIT  
Signed-off-by: .* <.*>**

 * Commit: a2ade40c9d21d001b795820793df176abbed5b81
  * Invalid signoff. Commit message must end with  
    **License: MIT  
Signed-off-by: .* <.*>**

[Guidelines](https://github.com/ipfs/community/blob/master/contribution-guidelines.md#commit-messages) and [a script](https://github.com/ipfs/community/blob/master/dev/tools/hooks/setup_commit_msg_hook.sh) are available to help. Your feedback on GitCop is welcome on [this issue](https://github.com/ipfs/infrastructure/issues/23).

---
This message was auto-generated by https://gitcop.com

**Comment by diasdavid** [comment]
should be all good now (squashed with the sig on https://github.com/diasdavid/go-ipfs/commit/a42daf2db9a8ce93835c874aacf1dcf4390d8b26)

File: core/commands/files/files.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ var log = u.Logger("cmds/files")
 
 var FilesCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Manipulate unixfs files",
+		Tagline: "Manipulate mfs files",
```
Comment (whyrusleeping) [inline core/commands/files/files.go]: I dont know that we actually want to call them 'mfs' files.. @jbenet?

Comment (jbenet) [inline core/commands/files/files.go]: `ipfs unix` or `ipfs files` 






—

**Comment by whyrusleeping** [comment]
@diasdavid ping!

**Comment by diasdavid** [comment]
@whyrusleeping pong! not sure what I am missing. I squashed all of the tiny commits into one and signed it https://github.com/diasdavid/go-ipfs/commit/a42daf2db9a8ce93835c874aacf1dcf4390d8b26 , isn't that good?

**Comment by whyrusleeping** [comment]
@diasdavid theres a bit of feedback on the phrasing of the helptext. we dont think it should be referred to as mfs

**Comment by jbenet** [comment]
it should be `ipfs unix` or `ipfs files`

**Comment by rht** [comment]
This should be rewritten against master; can be closed.

**Pull request closed without merging**